src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/Modal/feedback-affected-students-modal.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="modal-header">
+    <h4 class="modal-title" [jhiTranslate]="TRANSLATION_BASE + '.header'"></h4>
+    <button type="button" class="btn-close" aria-label="Close" (click)="activeModal.dismiss()"></button>
+</div>
+<div class="modal-body">
+    <table class="table table-striped mb-3">
+        <colgroup>
+            <col class="col" />
+            <col class="col" />
+            <col class="col" />
+        </colgroup>
+        <thead>
+            <tr>
+                <th [jhiTranslate]="TRANSLATION_BASE + '.name'"></th>
+                <th [jhiTranslate]="TRANSLATION_BASE + '.login'"></th>
+                <th [jhiTranslate]="TRANSLATION_BASE + '.repository'"></th>
+            </tr>
+        </thead>
+        <tbody class="table-group-divider">
+            @for (item of participation().content; track item) {
+                <tr>
+                    <td>{{ item.firstName }} {{ item.lastName }}</td>
+                    <td>{{ item.login }}</td>
+                    <td>
+                        <jhi-code-button
+                            class="ms-2"
+                            [smallButtons]="true"
+                            [repositoryUri]="item.repositoryURI"
+                            [routerLinkForRepositoryView]="['/courses', item.courseId, 'exercises', exerciseId(), 'repository', item.participationId]"
+                        />
+                    </td>
+                </tr>
+            }
+        </tbody>
+    </table>
+    <div class="d-flex flex-column align-items-end mt-2">
+        <ngb-pagination [collectionSize]="collectionsSize()" [pageSize]="pageSize()" [page]="page()" (pageChange)="setPage($event)" size="sm" [maxSize]="5"></ngb-pagination>
```

Comment:
Can we hide the pagination if we have fewer items than the maximum number that fits on one page? In other words, can we hide it, if the number of items fits on a single page and we don’t need an additional one.